roles/logrotate/tasks/main.yml

```diff
@@ -29,8 +29,8 @@
     group: root
   when: item.1
   with_together:
-    - [ mesos, docker, zookeeper ]
-    - [ true, true, role == 'control' ]
```

Comment:
Because vault SSH helper is installed via the vault role, e.g. only on control nodes